bindings/smtp/smtp.go

```diff
@@ -160,5 +174,29 @@ func (metadata Metadata) mergeWithRequestMetadata(req *bindings.InvokeRequest) M
 		merged.Subject = subject
 	}
 
-	return merged
+	if priority := req.Metadata["priority"]; priority != "" {
+		err := merged.parsePriority(priority)
+		if err != nil {
+			return merged, err
+		}
+	}
+
+	return merged, nil
+}
+
+func (metadata *Metadata) parsePriority(req string) error {
+	if req == "" {
+		metadata.Priority = defaultPriority
+	} else {
+		priority, err := strconv.Atoi(req)
+		if err != nil {
+			return err
+		}
+		if priority < 1 || priority > 5 {
```

Comment:
could add comment for priority?

Reply:
I added lowest and highest priority as const in order to be more clear.

Reply:
I am not familiar with SMTP priority, please share a doc here or comment in the code.

Reply:
https://www.chilkatsoft.com/p/p_471.asp